eng/Version.Details.xml

```diff
@@ -234,5 +234,9 @@
       <Uri>https://github.com/dotnet/roslyn-analyzers</Uri>
       <Sha>9a1f02eff42635a5a9934735f12c722c3bfba8e8</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Compatibility" Version="1.0.0-rc.1.21419.13">
```

Comment:
huh? I was expecting you to have to also remove a PackageValidation entry here. Does that mean that perhaps we were never updating the package as the full subscription wasn't set?

Reply:
PackageValidation hadn't had a subscription before. Its version was static.